src/prpl.ts

```diff
@@ -26,6 +26,14 @@ export interface Config {
   // Defaults to `max-age=60`.
   cacheControl?: string;
 
+  // A custom error-handling logic function. The function must takes two 
+  // arguments, request and response, and must return a function with
+  // the argument error.
+  //
+  // This function could be useful if you want, by example, be able to
+  // render custom errors pages.
+  error?: Function;
```

Comment:
You can declare this function signature more concretely, e.g. `(request: http.IncomingMessage, response: http.ServerResponse, error: ???) => void`. I'm not sure what the type of `error` should be

Reply:
I think it should be a `HttpError` from the `http-errors` module (module used by send).